packages/cli/src/commands/doctor/doctor.ts

```diff
@@ -108,23 +107,24 @@ export default (async function runDoctor(
             : undefined,
         };
       }),
-    )).filter(Boolean),
+    )).filter(healthcheck => healthcheck !== undefined) as HealthCheckResult[],
```

Comment:
Thought this would be a bit simpler to understand.
The `as HealthCheckResult[]` part is unfortunately needed, otherwise TypeScript isn't smart enough to know that the objects inside the array can't be `undefined` anymore.

Could have used [type predicates](https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-type-predicates) though.